assets/tests/gameRound.test.js

```diff
@@ -0,0 +1,57 @@
+import { GameBoard, GameRound } from '../js/app';
+import { displayController } from '../js/dom';
+
+const newGameRound = GameRound();
+newGameRound.player1.arr = [2, 3, 4];
+newGameRound.player2.arr = [0, 1, 2];
+
+
+describe('resetting of player arrays', () => {
+  test('it ensures playerArray is not empty', () => {
+    expect(newGameRound.player1.arr).not.toEqual([]);
+    expect(newGameRound.player2.arr).not.toEqual([]);
+  });
+
+  test('it tests if resetPlayerArrays works', () => {
+    newGameRound.resetPlayerArrays();
+    expect(newGameRound.player1.arr).toEqual([]);
+    expect(newGameRound.player2.arr).toEqual([]);
+  });
+});
+
+describe('checking if game is a draw', () => {
+  test('if isDraw returns false for moveCount less than 8', () => {
+    newGameRound.moveCount = 3;
+    expect(newGameRound.isDraw()).toBe(false);
+  });
+
+  test('if isDraw returns true for moveCount greater than 8', () => {
+    newGameRound.moveCount = 9;
+    expect(newGameRound.isDraw()).toBe(true);
+  });
+});
+
+describe('checking if current player can be obtained', () => {
+  test('if getCurrentPlayer return player 2 for even numbers', () => {
```

Comment:
* [ ]  Let us try to capture positive and negative scenarios for this test. 
```if getCurrentPlayer return player 2 for even numbers``` the other side is ```if getCurrentPlayer return player 1 for odd numbers```